### ISSUE TITLE
Add exception list or temporary deactivation

### ISSUE THREAD

**Issue by Globolo001** [oblador/hush#198]
I know this is agains the "no configuration" part. 
But I seriously have to switch to other browsers, to use certain websites. I would like to stick to safari. Thanks :)

**Comment by gingerbeardman** [comment]
Did you try "turn off content blockers" in current page options?

**Comment by Globolo001** [comment]
That will also deactivate other blockers (like adblocker) but is a somewhat acceptable workaround. Thanks!

**Comment by gingerbeardman** [comment]
Yeah, it is only workaround. I would also like the feature you proposed.

I'm using 1Blocker that does have such a customisation feature and am contrasting with Hush.
